GuildWars2-Calc/src/main/java/com/theironyard/services/GuildWar2CalcService.java

```diff
@@ -0,0 +1,52 @@
+package com.theironyard.services;
+
+import com.theironyard.entities.Item;
+import com.theironyard.entities.Price;
+import com.theironyard.entities.Recipe;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.RestTemplate;
+
+import java.util.List;
+
+/**
+ * Created by EddyJ on 7/25/16.
+ */
+@Service
+public class GuildWar2CalcService {
+
+    @Autowired
+    RestTemplate restTemplate;
+
+    public static final String RECIPE_URL = "https://api.guildwars2.com/v2/recipes/";
+
+    public static final String ITEM_URL = "https://api.guildwars2.com/v2/items/";
+
+    public static final String PRICE_URL = "https://api.guildwars2.com/v2/commerce/prices/";
+
+
+    public Recipe getByRecipe(Integer id){
+        Recipe recipe = restTemplate.getForObject(RECIPE_URL + id, Recipe.class);
+        return recipe;
+    }
+
+    public Item getByOutputItem(Integer id){
+        Item item = restTemplate.getForObject(ITEM_URL + id, Item.class);
+        return item;
+    }
+
+    public Item getIngredientList(Recipe recipe){
```

Comment:
Given that this method is making several calls to the api to populate the names and such I would expect there to be some level of documentation.